src/build/__tests__/build-test.js

```diff
@@ -149,4 +168,16 @@ describe('build()', () => {
       Timestamp: { serialize },
     }, undefined, false)).forEach((type, i) => expectTypesEqual(type, [Record, Timestamp][i]));
   });
+
+  it('should allow directives configuration using __schema', () => {
+    const config = { __schema: { directives: [customDirective] } };
+    const target = build(schemaSource + querySourceWithDirective, config, undefined, false);
+    expectSchemasEqual(target, SchemaWithDirective);
+  });
+
+  it('should not build schema with default directives', () => {
```

Comment:
I'm not entirely sure what this test case is testing for. Would you mind explaining what it's doing that the other test case doesn't catch? 😃